seed/lib/mappings/data/pm-mapping.json

```diff
@@ -458,6 +521,15 @@
     "type": "string",
     "schema": ""
   },
+  {
+    "display_name": "State Province",
+    "to_field": "state_province",
+    "to_table_name": "PropertyState",
+    "from_field": "State",
+    "units": "",
+    "type": "string",
+    "schema": ""
+  },
```

Comment:
I'm wondering if this (as well as the two above) should have a `to_field` of just `state` -- `state_province` isn't a real SEED column from what I can see on the PropertyState model.
@nllong do you know why we originally had it mapping to state_province?

Reply:
I agree. I think this should be just `"to_field": "state"`

Reply:
agreed